text/0000-style-rfcs.md

```diff
@@ -0,0 +1,323 @@
+
+If Rustfmt is to become a widely accepted tool, there needs to be a process for
+the Rust community to decide on the default style, and how configurable that
+style should be.
+
+These discussions should happen in the open and be highly visible. It is
+important that the Rust community has significant input to the process. The RFC
+repository would be an ideal place to have this discussion because it exists to
+satisfy these goals, and is tried and tested. However, the discussion is likely
+to be a high-bandwidth one (code style is a contentious and often subjective
+topic, and syntactic RFCs tend to be the highest traffic ones). Therefore,
+having the discussion on the RFCs repository could easily overwhelm it and make
+it less useful for other important discussions.
+
+There currently exists a [style guide](https://github.com/rust-lang/rust/tree/master/src/doc/style)
+as part of the Rust documentation. This is far more wide-reaching than just
+formatting style, but also not detailed enough to specify Rustfmt. This was
+originally developed in its [own repository](https://github.com/rust-lang/rust-guidelines),
+but is now part of the main Rust repository. That seems like a poor venue for
+discussion of these guidelines due to visibility.
+
+
+# Detailed design
+[design]: #detailed-design
+
+## Process
+
+The process for style RFCs will mostly follow the [process for other RFCs](https://github.com/rust-lang/rfcs).
+Anyone may submit an RFC. An overview of the process is:
+
+* If there is no single, obvious style, then open a GitHub issue on the
+  fmt-rfcs repo for initial discussion. This initial discussion should identify
+  which Rustfmt options are required to enforce the guideline.
+* Implement the style in rustfmt (behind an option if it is not the current
+  default). In exceptional circumstances (such as where the implementation would
+  require very deep changes to rustfmt), this step may be skipped.
+* Write an RFC formalising the formatting convention and referencing the
+  implementation, submit as a PR to fmt-rfcs. The RFC should include the default
+  values for options to enforce the guideline and which non-default options
+  should be kept.
+* The RFC PR will be triaged by the style team and either assigned to a team
+  member for [shepherding](https://github.com/rust-lang/rfcs#the-role-of-the-shepherd),
+  or closed.
+* When discussion has reached a fixed point, the RFC PR will be put into a final
+  comment period (FCP).
+* After FCP, the RFC will either be accepted and merged or closed.
+* Implementation in Rustfmt can then be finished (including any changes due to
+  discussion of the RFC), and defaults are set.
+
+
+### Scope of the process
+
+This process is specifically limited to formatting style guidelines which can be
+enforced by Rustfmt with its current architecture. Guidelines that cannot be
+enforced by Rustfmt without a large amount of work are out of scope, even if
+they only pertain to formatting.
```

Comment:
I considered a broader focus for the style team - including lints and unenforceable conventions. However, after some discussion, we thought it would be better to keep the style team as focussed as possible in order to get a quick resolution and to be really clear about the scope of the process.